### PR TITLE
Support for PNG player decals

### PR DESCRIPTION
**Pull request by a1batross**
- [x] Tested in local play
- [ ] Tested in network play

Resolves https://github.com/FWGS/xash3d-fwgs/issues/1127 

Needs server update, as customization is validated on server side as well.

Can be easily abused, needs texture resolution limit (before decoding, as it can be heavy).

### CODE REVIEW

File: engine/client/cl_main.c [hunk 1]
```diff
@@ -1206,38 +1207,44 @@ resource_t *CL_AddResource( resourcetype_t type, const char *name, int size, qbo
 
 void CL_CreateResourceList( void )
 {
-	char		szFileName[MAX_OSPATH];
+	char szFileName[MAX_OSPATH];
```
Comment (SNMetamorph) [inline engine/client/cl_main.c:1210]: Wrong identation

**Comment by SNMetamorph** [comment]
Add fix to mainui from #1130 also.

**Comment by a1batross** [comment]
@SNMetamorph this is out of scope but I fixed it. 

And of course rebased on top of png-decals branch, so PNG support is already in master.

Well, I don't see anything wrong with it anyway.

**Comment by a1batross** [comment]
Was merged to master already but GitHub didn't catch it :(

**Pull request closed without merging**